client/my-sites/stats/activity-log/index.jsx

```diff
@@ -264,12 +238,21 @@ class ActivityLog extends Component {
 			return applySiteOffset( moment.utc( ts_utc ) ).format( YEAR_MONTH ) === selectedMonthAndYear;
 		} );
 
+		if ( isEmpty( logsForMonth ) ) {
+			return (
+				<EmptyContent
+					title={ translate( 'No activity for %s', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **19** times:
`translate( 'Activity for {{period/}}' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).